docs/content/concepts/cryptography/nautilus/using-nautilus.mdx

```diff
@@ -111,7 +108,7 @@ Frontend code is not included in this guide. The Move call is demonstrated using
 
     :::
 
-1. To run the weather example as-is, you don't need to modify `allowed_endpoints.yaml` because it already includes `api.weatherapi.com`. Follow the prompts to enter the required values. This step demonstrates how to store a secret (an API key) using AWS Secrets Manager, so you can avoid including the secret in the public application code.
+##Step 1. To run the weather example as is, you don't need to modify `allowed_endpoints.yaml` because it already includes `api.weatherapi.com`. Follow the prompts to enter the required values. This step demonstrates how to store a secret (an API key) using AWS Secrets Manager, so you can avoid including the secret in the public application code.
```

Comment:
This feature isn't available yet. It's part of the indexer PR.
```suggestion
1. To run the weather example as is, you don't need to modify `allowed_endpoints.yaml` because it already includes `api.weatherapi.com`. Follow the prompts to enter the required values. This step demonstrates how to store a secret (an API key) using AWS Secrets Manager, so you can avoid including the secret in the public application code.
```